src/dcd/common/constants.d

```diff
@@ -20,187 +20,986 @@ module dcd.common.constants;
+	ConstantCompletion("isOverrideFunction", `$(P Takes one argument. If that argument is a function marked with
+	$(D_KEYWORD override), $(D true) is returned, otherwise $(D false).
+)
+
+---
+import std.stdio;
+
+class Base
+{
+	void foo() { }
+}
+
+class Foo : Base
+{
+	override void foo() { }
+	void bar() { }
+}
+
+void main()
+{
+	writeln(__traits(isOverrideFunction, Base.foo)); // false
+	writeln(__traits(isOverrideFunction, Foo.foo));  // true
+	writeln(__traits(isOverrideFunction, Foo.bar));  // false
+}
+---`),
+	ConstantCompletion("isPOD", `$(P Takes one argument, which must be a type. It returns
+$(D true) if the type is a $(DDSUBLINK glossary, pod, POD) type, otherwise $(D false).)`),
+	ConstantCompletion("isRef", isLazyDoc),
+	ConstantCompletion("isSame", `$(P Takes two arguments and returns bool $(D true) if they
+	are the same symbol, $(D false) if not.)
+
+---
+import std.stdio;
+
+struct S { }
+
+int foo();
+int bar();
+
+void main()
+{
+	writeln(__traits(isSame, foo, foo)); // true
+	writeln(__traits(isSame, foo, bar)); // false
+	writeln(__traits(isSame, foo, S));   // false
+	writeln(__traits(isSame, S, S));     // true
+	writeln(__traits(isSame, std, S));   // false
+	writeln(__traits(isSame, std, std)); // true
+}
+---
+
+$(P If the two arguments are expressions made up of literals
+or enums that evaluate to the same value, true is returned.)`),
+	ConstantCompletion("isScalar", `Works like $(D isArithmetic), except it's for scalar types.`),
+	ConstantCompletion("isStaticArray", `Works like $(D isArithmetic), except it's for static array types.`),
+	ConstantCompletion("isStaticFunction", `Takes one argument. If that argument is a static function,
+meaning it has no context pointer,
+$(D true) is returned, otherwise $(D false).`),
+	ConstantCompletion("isTemplate", `$(P Takes one argument. If that argument is a template then $(D true) is returned,
+	otherwise $(D false).
+)
+
+---
+void foo(T)(){}
+static assert(__traits(isTemplate,foo));
+static assert(!__traits(isTemplate,foo!int()));
+static assert(!__traits(isTemplate,"string"));
+---`),
+	ConstantCompletion("isUnsigned", `Works like $(D isArithmetic), except it's for unsigned types.`),
+	ConstantCompletion("isVirtualFunction", `The same as $(GLINK isVirtualMethod), except
+that final functions that don't override anything return true.`),
+	ConstantCompletion("isVirtualMethod", `Takes one argument. If that argument is a virtual function,
+$(D true) is returned, otherwise $(D false).
+Final functions that don't override anything return false.`),
+	ConstantCompletion("parent", `Takes a single argument which must evaluate to a symbol.
+The result is the symbol that is the parent of it.`)
 ];
 
 /**
  * Scope conditions
  */
-immutable string[] scopes = [
-	"exit",
-	"failure",
-	"success"
+immutable ConstantCompletion[] scopes = [
+	ConstantCompletion("exit", "executes statements when the scope exits normally or due to exception unwinding"),
```

Comment:
Some of the other completions are capitalized. Either these should be as well, or the others should be made lower-case.